### ISSUE TITLE
Issue on advantage point.

### ISSUE THREAD

**Issue by minimumnz** [xavikortes/ATP-Probability#1]
Hello, nice library.

I'm not quite sure how you're representing an advantage point. 

If I set score like the following:

Main.setPlayersAverage(0.65, 0.62)
Main.setScore(3, True, 1, 1, 1, 1, 4, 3, "A")
Main.getProbability("winMatch")

It has an error:
RecursionError: maximum recursion depth exceeded while calling a Python object

I had a look in the code and the error seems to be in 

from lib import Game
Game.probability( "A", 4, 3)

It's never exiting

if prob == -1:

because of this line:

prob = pointProb * probability(Server.keep(server), pointsA + 1, pointsB) + (1 - pointProb) * probability(Server.keep(server), pointsA, pointsB + 1)

if player A wins a point it just keeps incrementing the point score and not setting score to 0 which would exit that block.

**Comment by minimumnz** [comment]
I think adding these elif blocks to Game.probability fix the issue:

        elif pointsA == 4 and pointsB == 4:
            pointsA, pointsB = 3, 3
            prob = Deuce.probability(Server.keep(server))
        elif pointsA == 5 and pointsB == 3:
            prob = 1
        elif pointsA == 3 and pointsB == 5:
            prob = 0

**Comment by minimumnz** [comment]
I guess you are not using 4, 3 to represent advantage but going back to 3,2 since they're equivalent.

**Comment by xavikortes** [comment]
Yes, I'm going back to 3-2. I think this is not a bug, but for the direct 4-3 input, it can be solve by changing:

if pointsA == 3 and pointsB == 3: -> if pointsA == pointsB and pointsA >= 3:
